### DISCUSSION TITLE
How to set label "custom-kubelet=small-pods" to worker node during the cluster installation

### DISCUSSION THREAD

**Discussion by dbenduga** [okd-project/okd#1822] [General]
Hi,

Is there any way to set the label "custom-kubelet=small-pods" on the worker node in the install-config.yaml file?
Also, can we set the mem and CPU for the worker and master nodes in install-config.yaml?

**Comment by vrutkovs** [comment]
See https://docs.openshift.com/container-platform/4.14/installing/installing_aws/installing-aws-network-customizations.html on how to make modifications to installer manifests and https://docs.openshift.com/container-platform/4.14/machine_management/creating_machinesets/creating-machineset-aws.html for Machine API details
